### PR TITLE
Set Workflow definition id format to UUID

### PR DESCRIPTION
**Pull request by masayag**
**What this PR does / why we need it**:
As part of validating the correctness of API parameters, both requests and responses are updated for the workflow definition resource.

**Which issue(s) this PR fixes** :
Partially Fixes #[FLPATH-333](https://issues.redhat.com/browse/FLPATH-333)

**Change type**
- [ ] New feature
- [ ] Bug fix
- [ ] Unit tests
- [ ] Integration tests
- [ ] CI
- [ ] Documentation
- [x] Auto generated SDK code

**Impacted services**
- [x] Workflow Service
- [ ] Notifivcation Service

**Checklist**
- [x] Subject and description added to both, commit and PR.
- [x] Relevant issues have been referenced.

### CODE REVIEW

**Review by pkliczewski — COMMENTED** [review]
/lgtm

File: workflow-service/src/main/java/com/redhat/parodos/workflow/definition/service/WorkFlowDefinitionServiceImpl.java [hunk 1]
```diff
@@ -239,7 +239,7 @@ private void getWorksFromWorkDefinition(List<WorkFlowWorkDefinition> workFlowWor
 				return;
 			}
 			switch (workType) {
-				case TASK:
+				case TASK -> {
```
Comment (gciavarrini) [inline workflow-service/src/main/java/com/redhat/parodos/workflow/definition/service/WorkFlowDefinitionServiceImpl.java:242]: I like this :)

**Review by gciavarrini — COMMENTED** [review]
/lgtm

File: workflow-examples/src/test/java/com/redhat/parodos/examples/integration/SimpleWorkFlow.java [hunk 2]
```diff
@@ -91,8 +92,8 @@ public void runSimpleWorkFlow() throws ApiException, InterruptedException {
 		// Define WorkRequests
 		WorkRequestDTO work1 = new WorkRequestDTO();
 		work1.setWorkName("restCallTask");
-		work1.setArguments(
-				Arrays.asList(new ArgumentRequestDTO().key("url").value("http://localhost:8080/actuator/health")));
+		work1.setArguments(Collections
+				.singletonList(new ArgumentRequestDTO().key("url").value("http://localhost:8080/actuator/health")));
```
Comment (eloycoto) [inline workflow-examples/src/test/java/com/redhat/parodos/examples/integration/SimpleWorkFlow.java:96]: why this change?

**Review by eloycoto — APPROVED** [review]

File: workflow-examples/src/test/java/com/redhat/parodos/examples/integration/SimpleWorkFlow.java (hunk 2 above)
Comment (masayag) [reply workflow-examples/src/test/java/com/redhat/parodos/examples/integration/SimpleWorkFlow.java:96]: IDE magic. I'll revert :-)
